go.sum

```diff
@@ -43,6 +53,8 @@ github.com/docker/libtrust v0.0.0-20150114040149-fa567046d9b1 h1:ZClxb8laGDf5arX
 github.com/docker/libtrust v0.0.0-20150114040149-fa567046d9b1/go.mod h1:cyGadeNEkKy96OOhEzfZl+yxihPEzKnqJwvfuSUqbZE=
 github.com/felixge/httpsnoop v1.0.1 h1:lvB5Jl89CsZtGIWuTcDM1E/vkVs49/Ml7JJe07l8SPQ=
 github.com/felixge/httpsnoop v1.0.1/go.mod h1:m8KPJKqk1gH5J9DgRY2ASl2lWCfGKXixSwevea8zH2U=
+github.com/form3tech-oss/jwt-go v3.2.2+incompatible h1:TcekIExNqud5crz4xD2pavyTgWiPvpYe4Xau31I0PRk=
+github.com/form3tech-oss/jwt-go v3.2.2+incompatible/go.mod h1:pbq4aXjuKjdthFRnoDwaVPLA+WlJuPGy+QneDUgJi2k=
```

Comment:
I noticed this on the version for 2.7; https://github.com/distribution/distribution/pull/3465/files#r679126223

> This looks to be pulling in v3.2.2, which is missing a security fix in v3.2.4; [form3tech-oss/jwt-go@v3.2.2...v3.2.4](https://github.com/form3tech-oss/jwt-go/compare/v3.2.2...v3.2.4) (see https://github.com/form3tech-oss/jwt-go/tree/v3.2.4)

To get a more current version, you can add a `//indirect` to the go.mod

Reply:
Thanks, I was just checking the same thing. I think in this branch as long as github.com/Azure/go-autorest/autorest still uses github.com/form3tech-oss/jwt-go then using an //indirect to bump it to 3.2.4 is good.